src/test/java/pro/sky/EmployeesCollectionHomework/EmployeeServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package pro.sky.EmployeesCollectionHomework;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import pro.sky.EmployeesCollectionHomework.Exceptions.EmployeeAlreadyAddedException;
+import pro.sky.EmployeesCollectionHomework.Exceptions.EmployeeNotFoundException;
+import pro.sky.EmployeesCollectionHomework.Exceptions.EmployeeStorageIsFullException;
+import pro.sky.EmployeesCollectionHomework.Service.EmployeeService;
+import pro.sky.EmployeesCollectionHomework.Service.ValidationService;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+@SpringBootTest
+public class EmployeeServiceTest {
+
+    private final EmployeeService employeeService = new EmployeeService(new ValidationService());
+    private final List<Employee> employees = List.of(
+            new Employee("Иван", "Иванов", 1, 50000),
+            new Employee("Петр", "Петров", 1, 60000),
+            new Employee("Билл", "Гейтс", 2, 10000000)
+    );
+
+
+    @AfterEach
+    public void AfterEach() {
```

Comment:
```suggestion
    public void afterEach() {
```
с маленькой буквы методы пишутся